Source/common/SNTConfigurator.m

```diff
@@ -101,6 +101,10 @@ @implementation SNTConfigurator
 static NSString *const kBlockedPathRegexKey = @"BlockedPathRegex";
 static NSString *const kBlockedPathRegexKeyDeprecated = @"BlacklistRegex";
 
+// TODO(markowsky): move these to sync server only.
+static NSString *const kMetricsFormat = @"ExportMetricsFormat";
+static NSString *const kMetricsFile = @"ExportMetricsFile";
+
```

Comment:
What do you think of making this a URL where file:// URLs are accepted? I could see other formats being added that export via HTTPS

Reply:
That makes a lot of sense. My initial thought was to have one option per output type since  we have the issue of pull vs. push for different metric systems, e.g. prometheus expects that like borgmon of old we're serving HTTP.  

I'll update the metric service code to treat it as a URL instead.